axelor-base/src/main/java/com/axelor/apps/base/db/repo/LanguageBaseRepository.java

```diff
@@ -0,0 +1,35 @@
+package com.axelor.apps.base.db.repo;
+
+import com.axelor.apps.base.db.Language;
+import com.axelor.apps.base.web.LanguageService;
+import com.axelor.db.JPA;
+import com.axelor.inject.Beans;
+import javax.persistence.EntityManager;
+
+public class LanguageBaseRepository extends LanguageRepository {
+  @Override
+  public Language save(Language entity) {
+    // Get the original object by EntityManager.
+    EntityManager em = JPA.em().getEntityManagerFactory().createEntityManager();
+    Language oldLanguageObject = em.find(Language.class, entity.getId());
+    LanguageService languageService = Beans.get(LanguageService.class);
```

Comment:
You should inject this service because you also use it in the remove method
Check AddressBaseRepository per example and how we inject AddressService in it, you can do the same